demo/full/scripts/controllers/Player.tsx

```diff
@@ -254,13 +262,66 @@ function Player(): JSX.Element {
     [playerModule],
   );
 
+  const handleContentConfigChange = useCallback((contentConfigArg: ContentConfig) => {
+    setContentConfig(contentConfigArg);
+  }, []);
+
+  const generatedURL = React.useMemo<null | string>(() => {
+    if (contentConfig === null) {
+      return null;
+    }
+    return generateURLForConfig({
+      contentConfig,
+      loadVideoConfig: loadVideoOpts,
+      playerConfig: playerOpts,
+      demoConfig: { reactiveURL: isReactiveURLEnabled },
+    });
+  }, [contentConfig, loadVideoOpts, playerOpts, isReactiveURLEnabled]);
+
+  const [sharableURL, setSharableURL] = React.useState("");
+  React.useEffect(() => {
+    // still update URL if the last change was editing "isReactiveURLEnabled"
+    // value, so the "isReactiveURLEnabled" can be stored in the URL
+    const isEnabled = previousIsReactiveURLEnabled || isReactiveURLEnabled;
```

Comment:
I don't get the `previousIsReactiveURLEnabled` case

Reply:
There is a different approach that can work too:
```js
  React.useEffect(() => {
    const parsedHash = parseHashInURL(decodeURI(location.hash));
    const { disableReactiveURL } = parsedHash || {};

    const shouldUpdateHref = isReactiveURLEnabled || !disableReactiveURL;
    if (generatedURL && shouldUpdateHref) {
      setSharableURL(generatedURL);
      window.location.href = generatedURL;
    }
  }, [generatedURL, isReactiveURLEnabled]);
```